doc/rados/operations/upmap.rst

```diff
@@ -1,52 +1,53 @@
 .. _upmap:
 
-Using the pg-upmap
-==================
+Using pg-upmap
+==============
 
-Starting in Luminous v12.2.z there is a new *pg-upmap* exception table
+In Luminous v12.2.z and later releases, there is a *pg-upmap* exception table
 in the OSDMap that allows the cluster to explicitly map specific PGs to
-specific OSDs.  This allows the cluster to fine-tune the data
-distribution to, in most cases, perfectly distributed PGs across OSDs.
+specific OSDs. This allows the cluster to fine-tune the data distribution to,
+in most cases, perfectly distributed PGs across OSDs.
 
-The key caveat to this new mechanism is that it requires that all
-clients understand the new *pg-upmap* structure in the OSDMap.
+However, there is an important caveat when it comes to this new feature: it
+requires all clients to understand the new *pg-upmap* structure in the OSDMap.
 
 Enabling
 --------
 
-New clusters will by default enable the `balancer module`. The cluster must only
-have Luminous (and newer) clients. You can turn the balancer off with:
+By default, new clusters enable the *balancer module*. The cluster cannot have
+any pre-Luminous clients.  To turn off the balancer, run the following command:
 
 .. prompt:: bash $
 
    ceph balancer off
 
-To allow use of the feature on existing clusters, you must tell the
-cluster that it only needs to support Luminous (and newer) clients with:
+To allow use of the new feature on an existing cluster, you must restrict the
+cluster to supporting only Luminous (and newer) clients.  To do so, run the
+following command:
 
 .. prompt:: bash $
 
    ceph osd set-require-min-compat-client luminous
 
-This command will fail if any pre-Luminous clients or daemons are
-connected to the monitors.  You can see what client versions are in
-use with:
+This command will fail if any pre-Luminous clients or daemons are connected to
+the monitors. To see which client versions are in use, run the following
+command:
 
 .. prompt:: bash $
 
    ceph features
```

Comment:
There's actually a subtlety here, due to the way `ceph features` works.    There are clients that report `jewel` despite being upmap-compatible.  Details on the list, but probably we don't have to dive into that here given the age of said releases.

Reply:
I'm going to make a note of this to myself, because (if I survive long enough and there's sufficient time), I'd like to create documents that support the curiosity of any reader, no matter how recondite their area of curiosity is, and this seems like a good candidate for such a document.